### DISCUSSION TITLE
2.4.8

### DISCUSSION THREAD

**Discussion by benhillis** [microsoft/WSL#12389] [General]
*  Don't delete terminal profile & start menu shortcut when a distribution is converted
*  Plumb stderr handle through to WSL1 processes for import / export / setversion
*  Update modern distros to support WSL1 (solves #12298)
*  Make wsl --install case insensitive with modern distros
*  Fix logic error causing wsl2.processors to always be defaulted to the maximum... (solves #12346)
*  Update Microsoft.RemoteDesktop.Client.MSRDC.SessionHost to version 1.2.5716

<hr /><em>This discussion was created from the release <a href='https://github.com/microsoft/WSL/releases/tag/2.4.8'>2.4.8</a>.</em>

**Comment by Morrigan-Ship** [comment]
whats hidden features?
what about bridge mode?

**Comment by astroboylrx** [comment]
Whenever an update is released, Windows Store terminates my WSL and updates it for me, regardless of any running processes. I'm using WSL2 for work for a long time. I don't think the update was this aggressive previously, at least before 2024.
Is there something I can do to prevent this? Thanks in advance.

Maybe it is not Windows Store who actually did the update? Maybe the Store just shows an update was done? What module performs the update?
I already turned off auto app updates in the Store settings and even enabled "Turn off Automatic Downloads and Install of updates" in the group policy.

```
WSL version: 2.4.8.0
Kernel version: 5.15.167.4-1
WSLg version: 1.0.65
MSRDC version: 1.2.5716
Direct3D version: 1.611.1-81528511
DXCore version: 10.0.26100.1-240331-1435.ge-release
Windows version: 10.0.22635.4655
```

**Comment by 0xwidya-eth** [comment]
worth it

**Comment by ZupoLlask** [comment]
When can we expect news about definitely moving away from kernel 5.15? TY

**Reply by ericktreichel** [reply]
We would appreciate a 6.xx kernel to get advantage of all improvements for new hardwares
